Misc/NEWS.d/next/Library/2021-07-28-13-10-17.bpo-44761.smjHBU.rst

```diff
@@ -0,0 +1,2 @@
+Use "__main__" instead of "typing" as ``__module__`` attribute at
+``typing.NewType``. Patch provided by Yurii Karabas.
```

Comment:
Since we haven't released any Python versions between NewType became a class and we changed this here, I'd skip the NEWS entry altogether.